plugins/pci-mei/fu-pci-mei-plugin.c

```diff
@@ -31,7 +31,7 @@ fu_pci_mei_plugin_to_string(FuPlugin *plugin, guint idt, GString *str)
 					  "PciDevice",
 					  fu_device_get_id(self->pci_device));
 	}
-	for (guint i = 0; i < 6; i++) {
+	for (guint i = 1; i <= 6; i++) {
```

Comment:
good catch! Can we use `i < G_N_ELEMENTS(hfs_cfg_addrs)` as the condition?

Reply:
Sure

Reply:
Well, no, wrong function. But we can probably use `i < G_N_ELEMENTS(self->hfsts_buf)`.

Reply:
Works. Updated.